sql/core/src/test/scala/org/apache/spark/sql/DataFrameJoinSuite.scala

```diff
@@ -335,12 +336,6 @@ class DataFrameJoinSuite extends QueryTest
           spark.range(50).write.saveAsTable(s"$dbName.$table1Name")
           spark.range(100).write.saveAsTable(s"$dbName.$table2Name")
 
-          // First, makes sure a join is not broadcastable
-          val plan = sql(s"SELECT * FROM $dbName.$table1Name, $dbName.$table2Name " +
-            s"WHERE $table1Name.id = $table2Name.id")
-            .queryExecution.executedPlan
-          assert(plan.collect { case p: BroadcastHashJoinExec => p }.isEmpty)
```

Comment:
Oh, is this pre-testing removed completely?

Reply:
Ah, I saw that your comment.
- https://github.com/apache/spark/pull/27935#discussion_r395349072

Reply:
Yea, I think the current test coverage is good enough.